tornado/iostream.py

```diff
@@ -1387,6 +1387,10 @@ def _do_ssl_handshake(self) -> None:
                 )
                 return self.close(exc_info=err)
             raise
+        except ssl.CertificateError as err:
```

Comment:
`CertificateError` is a sub-class of `SSLError` so I think it will be caught by the except block just above:

```
>>> try:
...     raise ssl.CertificateError("whatever")
... except ssl.SSLError:
...     print("got SSLError")
... except ssl.CertificateError:
...     print("got CertficateError")
... 
got SSLError
```
```

Reply:
This appears to have changed in Python 3.7. Before that the two error classes were distinct. So we should add a comment to this effect but the change is fine